configs/qa/roles/subscriptions.json

```diff
@@ -32,6 +32,9 @@
         {
           "permission": "subscriptions:organization:read"
         },
+        {
```

Comment:
You'll need to bump the `"version"` above, to `3` in all environment files for the seed changes to be picked up.

Reply:
I have made the required changes. Please review